src/app/portlets/dot-edit-page/layout/components/dot-theme-selector/dot-theme-selector.component.scss

```diff
@@ -0,0 +1,76 @@
+@import 'variables';
+
+$theme-item-dimension: 240px;
+
+::ng-deep {
+    dot-site-selector {
+        dot-searchable-dropdown .ui-button,
+        .site-selector__title {
+            font-size: $font-size-large;
+            min-width: 180px;
+        }
+    }
+
+    .ui-datagrid-content {
+        width: $theme-item-dimension * 4 + $basic-padding * 6;
+        height: $theme-item-dimension * 2 + $basic-padding * 4;
+        margin: 0 $basic-padding * 2;
+
+        .ui-g {
+            margin: 0;
+        }
+    }
+
+    .ui-datagrid-content .ui-widget-content.ui-g-12 {
```

Comment:
Do you need the `.ui-g-12`?

Reply:
yes there is no way to add a selector at this level in the `<p-dataGrid>`, this is to cover the scenario where there is no records